sdk/firstload/src/listeners/web-socket/index.ts

```diff
@@ -0,0 +1,125 @@
+import { createNetworkRequestId } from '@highlight-run/client/src/listeners/network-listener/utils/utils'
+import type { HighlightWebSocketWindow } from '@highlight-run/client/src/listeners/network-listener/utils/web-socket-listener'
+
+declare var window: HighlightWebSocketWindow
+
+const placeholderCallback = () => null
+
+export const initializeWebSocketListener = () => {
```

Comment:
as we learned with the next.js setup, we'll want to do something like the following to make sure the monkeypatch can't happen more than once

https://github.com/highlight/highlight/blob/9e898c677527fce49a0f4d49bbe18bc4383bf75c/sdk/firstload/src/listeners/fetch/index.ts#L12-L15